tests/test_SequenceManager.cc

```diff
@@ -529,3 +532,265 @@ TEST_CASE("SequenceManager: store_sequence() & load_sequence() - Empty sequence"
         REQUIRE(seq_deserialized.empty());
     }
 }
+
+
+TEST_CASE("SequenceManager: git repository", "[SequenceManager]")
+{
+    std::filesystem::path git_dir {"sequences_git"};
+
+    SECTION("Create and store sequence")
+    {
+        // clean surrounding
+        std::filesystem::remove_all(git_dir);
+
+        // init manager
+        SequenceManager manager{ git_dir};
+
+        // create sequence
+        Sequence seq{ "git Sequence 1",  SequenceName{ "git_sequence_1" }, UniqueId{ 0xabcdef123456 } };
+        Step step_01{Step::type_while};
+        step_01.set_label("while");
+        step_01.set_script("return i < 10");
+        seq.push_back(step_01);
+
+        // store sequence and make git commit
+        manager.store_sequence(seq);
+
+        // unable to reach repo debug functions without a new function in sequenceManager
+        // Therefore a standalone git handler has to be initialized
+        git::GitRepository repo{git_dir};
+
+        // check if commit was done
+        const std::string expected_msg = "change sequence:\n- Add sequence.lua from new sequence 'git_sequence_1[0000abcdef123456]'\n- Add step_1_while.lua from new sequence 'git_sequence_1[0000abcdef123456]'";
```

Comment:
Line too long (style guide: max. 90 characters).

Reply:
I wanted to comment that this 90 chars limit is too short anyhow, but this line is indeed longer than the rattlesnakes on SLACs hill :-D

Reply:
```suggestion
        const std::string expected_msg = "change sequence:\n"
            "- Add sequence.lua from new sequence 'git_sequence_1[0000abcdef123456]'\n"
            "- Add step_1_while.lua from new sequence 'git_sequence_1[0000abcdef123456]'";
```